templates/default/opsmatic-agent.conf.erb

```diff
@@ -0,0 +1,12 @@
+# automatically generated by chef cookbook opsmatic-cookbook
+# additional info -> https://opsmatic.com/app/docs/agent-configuration
+
+<% if node["opsmatic"]["host_alias"] && !node["opsmatic"]["host_alias"].empty? %>
+# Host Alias
+host_alias = "<%= node["opsmatic"]["host_alias"] %>"
+<% end %>
+<% if node["opsmatic"]["groups"] && !node["opsmatic"]["groups"].empty? %>
```

Comment:
the attribute itself should be a list, and we should join that list on a comma in the template itself.